e2e/docker/swupdate/entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 
 openssl rand -hex 4 >/etc/device-id
 
-/usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
+/usr/bin/supervisord
```

Comment:
Why does one need supervisord?

Reply:
`supervisord` is used to run both `swupdate` and sample program that we are updating in tests, it's also handy to restart the services using `supervisorctl restart swupdate` without container is being stopped. In normal circumstances would be better to use `systemd` or some other alternative, but since we are running this inside `docker` I did not find a better solution.

Reply:
I see. In normal circumstances, you only want one service per container. But here that is fine, of course.